easybuild/toolchains/compiler/gcc.py

```diff
@@ -66,15 +69,19 @@ class Gcc(Compiler):
 
     # used when 'optarch' toolchain option is enabled (and --optarch is not specified)
     COMPILER_OPTIMAL_ARCHITECTURE_OPTION = {
-        systemtools.AMD : 'march=native',
-        systemtools.INTEL : 'march=native',
-        systemtools.POWER: 'mcpu=native',  # no support for march=native on POWER
+        (systemtools.AARCH32, systemtools.ARM): 'mcpu=native', # implies -march=native and -mtune=native
+        (systemtools.AARCH64, systemtools.ARM): 'mcpu=native', # since GCC 6; implies -march=native and -mtune=native
+        (systemtools.POWER, systemtools.POWER): 'mcpu=native',   # no support for -march on POWER; implies -mtune=native
+        (systemtools.X86_64, systemtools.AMD): 'march=native', # implies -mtune=native
+        (systemtools.X86_64, systemtools.INTEL): 'march=native', # implies -mtune=native
```

Comment:
@geimer this makes me wonder, should we also be using `-mcpu=native` on x86_84, since that implies both `-march=native` and `-mtune=native` for AARCH32?

Which one is closer to Intel's `-xHost`, `mcpu=native` or `-march=native`?